tools/replay_gen.py

```diff
@@ -176,6 +191,11 @@ def random_transaction(self):
             new_uuid[12:16] + '-' + new_uuid[16:20] + '-' + new_uuid[20:]
         transaction['all']['headers']['fields'].append(['uuid', new_uuid])
 
+        if has_yaml:
+            for i in range(len(transaction['all']['headers']['fields'])):
+                transaction['all']['headers']['fields'][i] = flow_style_list(
+                    transaction['all']['headers']['fields'][i])
```

Comment:
I think enumerate might be helpful here:

```
for i, field in enumerate(transaction['all']['headers']['fields']):
    transaction['all']['headers']['fields'][i] = flow_style_list(field)
```

If that works, we'll want this for the other loops like this below too.